### PR TITLE
Feature 2.0/blacklist

### PR DESCRIPTION
**Pull request by jrtate13**
Adding blacklist functionality

### CODE REVIEW

File: lib/GhostLogger.js [hunk 1]
```diff
@@ -22,31 +25,31 @@ class GhostLogger {
    * @param {object} [data]
    */
   debug(msg, data) {
-    _log(this.logger, this.logger.debug, msg, data);
+    _log(this.logger, this.logger.debug, this.options,  msg, data);
```
Comment (njgerner) [inline lib/GhostLogger.js]: remove extra space

File: lib/GhostLogger.js [hunk 2]
```diff
@@ -22,31 +25,31 @@ class GhostLogger {
...
   }
 
   /**
    * @param {string} msg
    * @param {object} [data]
    */
   info(msg, data) {
-    _log(this.logger, this.logger.info, msg, data);
+    _log(this.logger, this.logger.info, this.options,  msg, data);
```
Comment (njgerner) [inline lib/GhostLogger.js]: remove extra space

File: lib/GhostLogger.js [hunk 3]
```diff
@@ -22,31 +25,31 @@ class GhostLogger {
...
   }
 
   /**
    * @param {string} msg
    * @param {object} [data]
    */
   notice(msg, data) {
-    _log(this.logger, this.logger.notice, msg, data);
+    _log(this.logger, this.logger.notice, this.options, msg, data);
```
Comment (njgerner) [inline lib/GhostLogger.js:44]: remove extra space

File: lib/GhostLogger.js (hunk 1 above)
Comment (pwerth) [inline lib/GhostLogger.js]: remove extra space

File: lib/GhostLogger.js (hunk 2 above)
Comment (pwerth) [inline lib/GhostLogger.js]: and here

**Review by njgerner — APPROVED** [review]

**Pull request merged**